proxy/src/control_plane/client/neon.rs

```diff
@@ -252,9 +252,9 @@ impl NeonControlPlaneClient {
 
             Ok(node)
         }
-        .map_err(crate::error::log_error)
+        .inspect_err(|e| tracing::debug!(error = ?e))
         // TODO: redo this span stuff
```

Comment:
remove?

Reply:
Ok, I'm not sure what we discussed we wanted to do here. Maybe what I did here.

Reply:
Done.